README.md

```diff
@@ -134,6 +155,11 @@ StatusCode = {
 };
 ```
 
+## Envoy configuration
+Scalar DLT server relies on a custom header called `rpc.status-bin` to share error metadata with the client. This means envoy need to be
+configured to expose this header to the client.
+Concretely, `rpc.status-bin` need to be added to the `expose-headers` field of the [cors configuration](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#envoy-api-msg-route-corspolicy).
```

Comment:
```suggestion
More specifically, `rpc.status-bin` needs to be added to the `expose-headers` field of the [cors configuration](https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#envoy-api-msg-route-corspolicy).
```

Reply:
@scalarindetail I can't find a description about `rpc.status-bin` in the referenced doc.
Can you specify the location more specifically ?

Reply:
Allow me to answer this question since I initially worked on that part of the code. 

I am afraid there is no official documentation yet explaining the modification needed on Envoy configuration. The documentation linked is the general envoy documentation and not a documentation how envoy should be configured to be used as grpc-web proxy. I could not find any official documentation on the grpc-web repository which mentions the modifications needed to the envoy configuration to white-list metadata headers. At the time, I was able to understand how it works by browsing some issues on the grpc-web repository and checking out the source code.

Below you can find an example of the changes needed to the envoy configuration used to run integration test on circle-ci. You can see that the header name `rpc.status-bin` was added to the list of `exposed-headers` field of the coors configuration.

https://github.com/scalar-labs/scalardl-web-client-sdk/blob/234bf930290eb03935bd8f84afb41010bcaef53d/.circleci/envoy.yaml#L28

Reply:
I think the "rpc.status-bin" name originates from this utility method that is used in the `AbstractLedgerClient` 
```
protected static final Key<Status> STATUS_TRAILER_KEY = ProtoUtils.keyForProto(Status.getDefaultInstance());
```

Reply:
Oh, OK, it seems very `gprc-java` thing  isn't it... Anyways thank  you !
I'm just wondering how you found it ? Just checked the response ?

Reply:
I found that the header was named `rpc.status-bin` by using the node-grpc. Since node-grpc do not need extra configuration to white-list metadata header like java-grpc. The error.metada field contained the single `rpc.status-bin` header.  
The key aspect was figuring out that the metadata field of the JS error thrown by the grpc web client are the http response headers. By looking at all the issue talking about the metadata, I found the issue that linked to a feature proposal adding the metadata support to JS error. 
https://github.com/grpc/grpc-web/pull/667
This line of code ``` metadata: responseHeaders``` gave me the answer I was looking for.
https://github.com/grpc/grpc-web/pull/667/files#diff-384bd19901f3116fd3698d5db6f1ba87R228